### DISCUSSION TITLE
Installing DeepMD on Windows 10

### DISCUSSION THREAD

**Discussion by ali90-prog** [deepmodeling/deepmd-kit#305] [Q&A] [answered]
Hello, 

I tried two different methods that you provided for installing DeepMD (the conda and the python interface methods). But, non of them worked properly. Have you tried to install it on Windows? What is the best method? 

Thanks,
Ali

**Comment by njzjz** [comment]
We suggest to use Windows Subsystem Linux (WSL). None of developers installed DP on Windows so we cannot provide help.

**Comment by njzjz** [comment, marked as answer]
Another choice is using Docker: https://github.com/orgs/deepmodeling/packages/container/package/deepmd-kit
